template/ios/carthage/PROJECT.playground/Pages/page.xcplaygroundpage/Contents.swift

```diff
@@ -1 +1 @@
-import Foundation
+import SwiftUI
```

Comment:
Why do you need to import SwiftUI here? It creates an unnecessary dependency.

Reply:
Yep we can remove the whole dependencies...